### PR TITLE
Move project to AIO Firefly structure and migrate to AIO CLI

### PR DESCRIPTION
**Pull request by mhaack**
## Description

Move project to AIO Firefly structure
Migrate to AIO CLI

## Motivation and Context

Align and prepare project for AEM Firefly

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code passes the code style as defined by ESLint.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      cart:
+        function: actions/remote/cartResolver.js
+        runtime: 'nodejs:14'
+        inputs:
+          LOG_LEVEL: info
+
+      # This deploys a GraphQL introspection endpoint that includes all the Magento types and fields
+      # used by the AEM/Magento integration.
+      cif-schema:
+        #name: graphql-reference/cif-schema
```
Comment (herzog31) [inline manifest.yml:30]: Is this line needed?

File: web-src/404.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<!-- <license header> -->
```
Comment (herzog31) [inline web-src/404.html:1]: All these `<license header>` placeholders should be replaced by an Apache license header.

**Review by herzog31 — APPROVED** [review]

File: web-src/404.html (hunk 2 above)
Comment (mhaack) [reply web-src/404.html:1]: This file is removed in the #78 PR

File: manifest.yml (hunk 1 above)
Comment (mhaack) [reply manifest.yml]: Nope, removed it.

**Pull request merged**